### PR TITLE
This works around a gcc-9 crash

### PR DESCRIPTION
**Pull request by bernd-edlinger**
It happens when configured with ./config -ftest-coverage

see: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=94029

Master is not affected by this issue.

### CODE REVIEW

**Comment by t8m** [comment]
LGTM

File: crypto/pem/pem_all.c [hunk 1]
```diff
@@ -84,9 +84,9 @@ IMPLEMENT_PEM_write_cb_const(RSAPrivateKey, RSA, PEM_STRING_RSA,
 
 
 IMPLEMENT_PEM_rw_const(RSAPublicKey, RSA, PEM_STRING_RSA_PUBLIC,
-                       RSAPublicKey) IMPLEMENT_PEM_rw(RSA_PUBKEY, RSA,
-                                                      PEM_STRING_PUBLIC,
-                                                      RSA_PUBKEY)
+                       RSAPublicKey)
+    IMPLEMENT_PEM_rw(RSA_PUBKEY, RSA, PEM_STRING_PUBLIC,
+                     RSA_PUBKEY)
```
Comment (levitte) [inline crypto/pem/pem_all.c]: It's still too indented...  it should be flush to the left

Comment (bernd-edlinger) [reply crypto/pem/pem_all.c]: the same funny indentation is in other places in this file too,
shall I fix them as well?

Comment (levitte) [reply crypto/pem/pem_all.c]: Please do

**Comment by bernd-edlinger** [comment]
The unusual formatting triggered an assertion, about the line numbers looked bogus.

**Comment by bernd-edlinger** [comment]
... and of course "line coverage" is pretty pointless, when we define multiple functions
on one macro in one line.
One might say these are a headache to debug as well.

**Comment by t8m** [comment]
Still approved

**Comment by bernd-edlinger** [comment]
Okay, merged to OpenSSL_1_1_1-stable bf42b99
Thanks!

**Pull request closed without merging**